### ISSUE TITLE
Fans work in sleep mode 

### ISSUE THREAD

**Issue by nofirg** [cheneyveron/clover-x79-e5-2670-gtx650#34]
Thx for publish your project, it help me to run MacOS on my i7-3820  + MB E5 3.5b
Could you please give me advice or forward me to right direction. 
I have problem with sleep mode: when computer going to sleep, fans still work in usual mode, only display switch to sleep mode. I suppose it can be fixed by enable s3 power mode, but I have no that option in my bios, only s1. Does it right thought? If yes, can I fix it by dsdt patch or may be modify bios rom?

**Comment by ichswarox** [comment]
no S3 power mode support on this motherboard

**Comment by nofirg** [comment]
And its main and irreparable reason sleep issue?

**Comment by ichswarox** [comment]
Yes, because huanan mother board not support S3 power management.
S1 - Stand By (Power On Suspend) - no system context is lost
S2 - Stand By - CPU and system cache context is lost 
S3 - Stand By (Suspend to Ram) - system memory context is maintained, all other system context is lost

**Comment by nofirg** [comment]
It's so sorry.
May be you know, this power management hardware or software implementation? May be it's possible to fool the system

**Comment by ichswarox** [comment]
https://github.com/cheneyveron/clover-x79-e5-2670-gtx650/issues/9

**Comment by cheneyveron** [comment]
S3 status of sleep requires BIOS/UEFI modification, and this is one of the hardest part of most X79 mobos which i haven't found a way out.
